### ISSUE TITLE
problem for reproducing the eval_loss curve when training reward model

### ISSUE THREAD

**Issue by tjlujianqiao** [CarperAI/trlx#447]
### 🐛 Describe the bug

As We use the same configuration in train_reward_model_gptj.py,
Specifically, 
```python
    training_args = TrainingArguments(
        report_to="none",
        output_dir="rm_checkpoint/",
        num_train_epochs=5,
        logging_steps=10,
        gradient_accumulation_steps=4,
        save_strategy="steps",
        evaluation_strategy="steps",
        per_device_train_batch_size=1,
        per_device_eval_batch_size=1,
        eval_accumulation_steps=1,
        eval_steps=500,
        save_steps=500,
        warmup_steps=100,
        logging_dir="./logs",
        fp16=True,
        bf16=False,
        learning_rate=1e-5,
        deepspeed="ds_config_gpt_j.json",
        save_total_limit=1,
    )
```

I cannot reproduce the same eval_loss curve announced at https://wandb.ai/carperai/summarize_RLHF/runs/1lmgvptq?workspace=user-, can anybody help me solve the problem?
